code-of-conduct.html

```diff
@@ -72,7 +72,8 @@ <h3 class="h3">Scope</h3>
             </div>
             <div class="pb-6">
                 <h3 class="h3">Enforcement and Reporting</h3>
-                <p> We encourage all communities to resolve issues on their own whenever possible. Instances of abusive, harassing, or otherwise unacceptable behavior may be reported to the community leaders responsible for enforcement at opencode@microsoft.com.</p>
+                <p> We encourage all communities to resolve issues on their own whenever possible. Instances of abusive, harassing, or otherwise unacceptable behavior should be reported to the community leaders responsible for enforcement in a given project or to opencode@microsoft.com. If you are a Microsoft employee looking for support, please use the <a href="https://aka.ms/community-911-landingpage">Community 911 reporting process</a>.
```

Comment:
should this link open in a new window or tab, with `target="_new"`, or self by default?

Reply:
updated